source/wp-content/themes/wporg-developer-2023/inc/autocomplete.php

```diff
@@ -108,13 +108,9 @@ public function autocomplete_data_update() {
 			wp_send_json_error( $defaults );
 		}
 
-		foreach ( $form_data['post_type'] as $key => $post_type ) {
-			if ( ! in_array( $post_type, $parser_post_types ) ) {
-				unset( $form_data['post_type'][ $key ] );
-			}
-		}
-
-		$post_types = ! empty( $form_data['post_type'] ) ? $form_data['post_type'] : $parser_post_types;
+		$post_types = isset( $_POST['post_type'] ) && 'command' === $_POST['post_type'] ?
+			array( 'command' ) :
+			$parser_post_types;
```

Comment:
Nitpick but I think this format (typically enforced by eslint) is clearer
```suggestion
		$post_types = isset( $_POST['post_type'] ) && 'command' === $_POST['post_type']
		        ? array( 'command' )
		        : $parser_post_types;
```